tests/testing_utils/fixtures.py

```diff
@@ -268,7 +268,7 @@ def temp_directory_for_app_zip(temp_dir) -> Generator:
     yield temp_dir.name
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
```

Comment:
Changed scope of unit tests' `test_snowcli_config` fixture from `session` to `function` to avoid side effects between tests.